### DISCUSSION TITLE
CatalogItems endpoint isn't providing us with the ASIN - EAN mapping details

### DISCUSSION THREAD

**Discussion by vteegapuram1** [amzn/selling-partner-api-models#3807] [General] [answered]
I have tried sending sp-api requests for search_catalog_items/get_catalog_item functions and able to see the ASIN related info without UPC/EAN as response from it. Please let us know if there is a way to get ASIN and it's relevent EAN/UPC from CatalogItems endpoint. Attaching the output response which I get without UPC/EAN.

![US_AMAZON_CATALOG_OUTPUT_RESPONSE](https://github.com/amzn/selling-partner-api-docs/assets/161826427/2dabc82f-9eab-4cb5-b3b1-a2a50544e1af)

**Comment by kevnhowe** [comment, marked as answer]
First, I'd like to say that this seems like a fairly active community, but the response levels are disappointingly low, especially for a relatively simple solution like this one...
It appears as if you have not passed an "IncludedData" array parameter in your request. The default value for not including the optional parameter is "summaries", which I see as the only data included your response.
The UPC/EAN data is returned when you request the "Identifiers" option in the IncludedData parameter.
Take a look at the Catalog API reference for details:
https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-reference#get-catalog2022-04-01items

**Reply by vteegapuram1** [reply]
Thanks alot, it worked.
